docs/guide/worker.md

```diff
@@ -43,6 +43,57 @@ Worker starting in daemon mode with `queue/listen` command supports [File], [Db]
 [Beanstalk]: driver-beanstalk.md
 [Gearman]: driver-gearman.md
 
+Systemd
+-------
+
+Systemd is a Linux system manager used to bootstrap daemons. To configure workers startup
+using systemd, create a config file named `yii-queue@.service` in `/etc/systemd/system` with
+the following contents:
+
+```conf
+[Unit]
+Description=Yii Queue Worker %I
+After=network.target
+After=mysql.service
+Requires=mysql.service
```

Comment:
add a comment here: "this only applies if your queue backend is mysql, replace this with the service that powers your backend."